app/scripts/views/settings/clients.js

```diff
@@ -137,33 +136,33 @@ define(function (require, exports, module) {
       }
     },
 
-    _onRefreshClientsList () {
-      if (this.isPanelOpen()) {
-        this.logViewEvent('refresh');
-        // only refresh devices if panel is visible
-        // if panel is hidden there is no point of fetching devices
-        this._fetchAttachedClients().then(() => {
-          this.render();
-        });
-      }
-    },
-
     _onGetApp (event) {
       var appType = this.$el.find(event.currentTarget).data('get-app');
       this.logViewEvent(`get.${appType}`);
     },
 
     openPanel () {
       this.logViewEvent('open');
-      this._fetchAttachedClients();
+      this.$el.find('.clients-refresh').trigger('submit');
```

Comment:
It's not clear why you would want to trigger a submit, 6 months from now we are going to look at this and go "WTF?" Can you leave a comment about why? Can the submit be triggered w/o using a DOM event handler, seems a round about way of doing it.